daemon/api.go

```diff
@@ -868,6 +880,7 @@ type snapInstructionResult struct {
 	summary  string
 	affected []string
 	tasksets []*state.TaskSet
+	apiData  map[string]interface{}
```

Comment:
we're currently setting api-data to the list of affected snaps; I want to add something to that, so I thought a nice way of doing it would be to return the data from the handler func

Reply:
I _could've_ gotten rid of `affected`, but figured the delta would've been quite a bit bigger. This way I can come back and do that change later.